src/System.CommandLine.Rendering.Tests/TerminalModeTests.cs

```diff
@@ -44,7 +44,7 @@ public async Task Sets_output_mode_to_Ansi_when_specified_by_output_directive(Ou
             OutputMode detectedOutputMode = OutputMode.Auto;
 
             var command = new Command("hello");
-            command.SetHandler((InvocationContext ctx) =>
+            command.SetHandler(ctx =>
```

Comment:
it's really nice for the simple handlers to be naturally-typed here (and in the other 1000 callsites in this PR)

Reply:
Agreed. Settling on `InvocationContext` for the case where we do this (i.e. removing support for `ParseResult`, etc. as parameters) reduces a lot of DI-ish complexity in these handlers. If we pull this thread hard enough, the need for the service provider might disappear.